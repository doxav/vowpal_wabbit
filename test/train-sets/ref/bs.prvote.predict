0.498689 0.458390 1.000000
0.452659 0.452659 0.619416
0.310507 0.310507 0.456509
0.204111 0.173337 0.336450
0.248373 0.248373 0.432724
1.000000 0.935124 1.000000
0.306509 0.283262 0.428032
0.190634 0.045960 0.367021
0.435073 0.355603 0.473136
1.000000 0.949092 1.000000
0.368406 0.191055 0.379315
0.324045 0.324045 0.538324
0.209115 0.191435 0.419371
0.310091 0.310091 0.535169
0.994895 0.902809 0.994895
0.885632 0.592456 0.885632
0.860310 0.860310 1.000000
0.124991 0.057171 0.353277
0.447352 0.190818 0.644261
0.169037 0.045328 0.169037
0.800762 0.230530 0.923944
0.997210 0.611789 1.000000
0.174761 0.070341 0.310366
1.000000 0.489735 1.000000
0.000000 0.000000 0.151716
0.475542 0.138823 0.475542
0.224301 0.224301 0.449844
0.044406 0.044406 0.248419
0.993207 0.468885 0.996748
0.542972 0.243104 0.542972
0.974816 0.605581 1.000000
0.428626 0.138691 0.428626
0.122979 0.122979 0.198036
0.416153 0.272718 0.450172
0.923723 0.685431 0.997519
0.175658 0.175658 0.570998
0.654157 0.654157 1.000000
0.099213 0.099213 0.276944
0.978369 0.602038 1.000000
1.000000 1.000000 1.000000
0.109665 0.109665 0.408097
0.826258 0.826258 0.946926
0.019578 0.011032 0.125593
0.147675 0.147675 0.375115
0.309596 0.034709 0.309596
0.083988 0.083988 0.181821
0.701049 0.119488 0.701049
0.068501 0.043627 0.557042
0.290648 0.290648 0.961409
0.257622 0.222167 0.287373
0.980525 0.428921 1.000000
1.000000 0.368642 1.000000
0.000000 0.000000 0.401585
0.216212 0.092099 0.481334
1.000000 0.969464 1.000000
0.222806 0.222806 1.000000
0.132120 0.119060 0.317708
0.380972 0.069827 0.380972
0.371972 0.371972 0.885179
0.067895 0.067895 0.437237
1.000000 0.570298 1.000000
0.020918 0.016143 0.348595
0.727962 0.597393 0.885829
0.091750 0.066054 0.201452
1.000000 0.621175 1.000000
0.109883 0.093998 0.313378
0.454445 0.069193 0.460195
0.137345 0.095883 0.806661
0.196326 0.045854 0.196326
0.566345 0.566345 0.797911
0.070047 0.056481 0.425846
0.654710 0.539467 0.902148
0.976362 0.283821 0.976362
0.000000 0.000000 0.127378
0.861996 0.726537 1.000000
1.000000 0.459947 1.000000
0.228147 0.029138 0.371215
0.187811 0.187811 0.601652
0.119670 0.057713 0.119670
0.209628 0.059909 0.347789
0.000000 0.000000 0.192228
0.141604 0.141604 0.400031
1.000000 0.787474 1.000000
0.102555 0.102555 0.489686
0.167892 0.150986 0.222639
0.117383 0.117383 0.415650
0.382087 0.345910 1.000000
1.000000 0.664915 1.000000
1.000000 0.772280 1.000000
0.058930 0.000000 0.300062
0.053952 0.053952 0.209100
0.929145 0.534163 0.951390
1.000000 1.000000 1.000000
0.158250 0.072309 0.175301
0.681810 0.681810 1.000000
0.002423 0.000000 0.213567
1.000000 0.504172 1.000000
0.319510 0.027735 0.319510
0.950059 0.384518 0.950059
0.985940 0.533635 1.000000
0.009065 0.000000 0.401651
0.534494 0.534494 0.939360
0.000000 0.000000 0.104174
1.000000 0.493532 1.000000
0.118083 0.000000 0.147888
1.000000 0.963237 1.000000
0.000000 0.000000 0.143924
0.415193 0.048358 0.415193
0.228804 0.133686 0.228804
0.995635 0.917059 1.000000
0.683965 0.665323 1.000000
0.628948 0.154025 0.628948
0.000000 0.000000 0.408536
1.000000 0.317683 1.000000
0.085899 0.000000 0.509912
0.020356 0.000000 0.506964
0.850782 0.323823 0.987185
0.452777 0.452777 1.000000
0.327837 0.327837 1.000000
0.060273 0.001862 0.496232
0.018962 0.018962 0.388772
0.949857 0.927328 0.949857
0.083230 0.000000 0.404609
0.934196 0.453135 1.000000
0.601575 0.495457 0.991451
0.369999 0.369999 0.885172
0.049708 0.000000 0.405252
0.449116 0.449116 0.886408
0.092203 0.000000 0.400786
0.554368 0.414330 0.945832
0.020886 0.000000 0.117271
1.000000 0.733348 1.000000
0.057783 0.037800 0.243426
0.237757 0.237421 0.939561
0.005307 0.000000 0.367407
0.231666 0.026962 0.300478
0.588162 0.496068 1.000000
0.973113 0.350595 0.995408
1.000000 0.954148 1.000000
0.379896 0.029820 0.379896
0.031756 0.002710 0.456161
0.000000 0.000000 0.174533
0.703112 0.683823 0.909649
0.916332 0.823318 0.990685
0.985697 0.408713 0.985697
0.611208 0.611208 0.920679
0.945687 0.488054 0.945687
1.000000 0.892140 1.000000
0.000000 0.000000 0.323415
0.909166 0.309824 0.909166
1.000000 0.908086 1.000000
0.231055 0.215659 0.913834
0.926225 0.622499 0.926225
0.000000 0.000000 0.364806
0.504040 0.000000 0.663357
0.901055 0.528311 0.920641
0.961164 0.937749 1.000000
0.061637 0.000000 0.061637
1.000000 0.849636 1.000000
0.000000 0.000000 0.621811
0.548149 0.548149 0.956069
0.000000 0.000000 0.203684
0.040461 0.040461 0.379248
0.781187 0.678339 0.936412
0.331340 0.000532 0.331340
0.409029 0.409029 0.923825
1.000000 0.933654 1.000000
0.086990 0.000000 0.086990
0.417787 0.417787 0.997241
1.000000 0.705726 1.000000
0.913484 0.584154 0.913484
0.301853 0.029493 0.334615
0.000000 0.000000 0.273973
0.446529 0.314658 0.826037
0.200166 0.045142 0.200166
0.478625 0.031250 0.597997
0.275946 0.025126 0.275946
0.971336 0.774050 0.971336
0.975631 0.677971 1.000000
0.738303 0.733083 0.959718
0.448380 0.448380 0.996360
0.148851 0.083232 0.174466
0.425506 0.425506 0.908774
0.005152 0.000000 0.248872
0.068516 0.068516 0.384858
0.032268 0.000000 0.513376
0.627244 0.627244 1.000000
0.558645 0.000000 0.558645
0.036497 0.036497 0.363344
0.994183 0.671142 0.994183
0.876763 0.661582 0.920168
0.114733 0.075627 0.462856
0.014072 0.000000 0.035419
0.297109 0.000000 0.297109
0.063500 0.063500 0.297037
1.000000 0.982524 1.000000
0.450480 0.450480 0.962684
0.009299 0.000000 0.019079
0.345340 0.018797 0.458421
1.000000 0.532585 1.000000
