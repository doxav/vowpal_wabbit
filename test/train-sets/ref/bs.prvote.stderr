only testing
Num weight bits = 18
learning rate = 10
initial_t = 1
power_t = 0.5
predictions = bs.prvote.predict
using no cache
Reading datafile = train-sets/0001.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
1.000000   1.000000            1         1.0   1.0000   0.4987       51
1.000000   1.000000            2         2.0   0.0000   0.4527      104
1.000000   1.000000            4         4.0   0.0000   0.2041      135
0.875000   0.750000            8         8.0   0.0000   0.1906      146
0.875000   0.875000           16        16.0   1.0000   0.8856       24
0.875000   0.875000           32        32.0   0.0000   0.4286       32
0.859375   0.843750           64        64.0   0.0000   0.0918       61
0.812500   0.765625          128       128.0   1.0000   0.4491      106

finished run
number of examples per pass = 200
passes used = 1
weighted example sum = 200
weighted label sum = 0
average loss = 0.805
best constant = -0.00502513
total feature number = 15482
